common/src/main/scala/models/Registration.scala

```diff
@@ -1,6 +1,6 @@
 package models
 
-case class Registration(deviceId: String, platform: Platform, userId: UserId, topics: Set[Topic])
+case class Registration(deviceId: String, platform: Platform, userId: UniqueDeviceIdentifier, topics: Set[Topic])
```

Comment:
It would be nice to, but that would be a breaking change for the windows 10 client